web/themes/new_weather_theme/assets/sass/_uswds-theme-custom-styles.scss

```diff
@@ -45,8 +45,17 @@ body {
   height: 100%;
 }
 
+
 /* Current Conditions
-------------------------------------- */
+-------------------------------------
+*/
 .current-conditions-temp {
   font-size: 56px;
 }
+  
+/* Hourly Forecast Block Overrides
+--------------------------------------
+*/
+.hourly-forecast-block ol > li {
+  @include u-margin-bottom(3);
+}
```

Comment:
Minor thing: we previously created some files at `assets/sass/components/` for individual component customizations. That may have been premature since there are so few of them. I don't think we need to do anything right now, but we may eventually want to pick one way or the other. I'm indifferent!

Reply:
Oh I must have forgot about this, will check it out when we refactor